ui/main/src/app/modules/feed/components/card-list/card-list.component.ts

```diff
@@ -74,26 +76,31 @@ export class CardListComponent implements AfterViewChecked, OnInit {
     }
 
     acknowledgeAllVisibleCardsInTheFeed() {
-        this.lightCards.forEach(lightCard => {
-
-            const processDefinition = this.processesService.getProcess(lightCard.process);
-
-            if (! lightCard.hasBeenAcknowledged && this.isCardPublishedBeforeAckDemand(lightCard)
-                && this.acknowledgeService.isAcknowledgmentAllowed(this.currentUserWithPerimeters, lightCard, processDefinition)) {
-                try {
-                    const entitiesAcks = [];
-                    const entities = this.entitiesService.getEntitiesFromIds(this.currentUserWithPerimeters.userData.entities);
-                    entities.forEach(entity => {
-                        if (entity.entityAllowedToSendCard) // this avoids to display entities used only for grouping
-                            entitiesAcks.push(entity.id);
-                    });
-                    this.acknowledgeService.acknowledgeCard(lightCard, entitiesAcks);
-                } catch (err) {
-                    console.error(err);
-                    this.displayMessage('response.error.ack', null, MessageLevel.ERROR);
-                }
+       this.lightCards.forEach(lightCard => {
+            this.acknowledgeVisibleCardInTheFeed(lightCard);
+            this.groupedCardsService.getChildCardsByTags(lightCard.tags).forEach(groupedCard =>
```

Comment:
Not necessary from a code or performance point-of-view. If the grouped-card function is not active, there will be no "child card' and thus getChildCardByTags always returns an empty array.

**Detail:**
The groupedChildCards and parentsOfGroupedCards arrays from grouped-card-service.ts are computed/updated when "computeGroupedCards" is called from the "computeFilteredAndSortedLightCards" from lightcards-feed-service.ts. If it is not called, groupedChildCards (and also parentsOfGroupedCards) will be an empty array.

However, if you still want a check because it makes it easier for developers to understand what is going on, I can add it..

Reply:
ok , you are right